src/runtime_src/core/edge/include/sk_types.h

```diff
@@ -30,7 +31,8 @@
  * freeBO    : free BO handle.
  * logMsg    : send log messages to XRT driver for saving as per ini settings
  */
-struct sk_operations {
+struct XRT_DEPRECATED
```

Comment:
Should the `XRT_DEPRECATED` come before `struct`?

Reply:
I had it before struct, but got warning when compiling saying attribute ignored, and it need to follow struct keyword

Reply:
Thanks for detail.